src/mantine-dates/src/components/TimeInput/demos/disabled.tsx

```diff
@@ -8,7 +8,7 @@ const code = `
 function Demo() {
   return (
     <div style={{ maxWidth: 320, marginLeft: 'auto', marginRight: 'auto' }}>
-      <TimeInput label="Pick time" disabled />
+      <TimeInput label="Pick time" disabled defaultValue={new Date()} />
```

Comment:
- Inputs that go after each other in demos must have `mt="md"`
- Change inputs labels to name features: "Disabled without value" and "Disabled with value"